### ISSUE TITLE
line_break_multiple_parameters too strict

### ISSUE THREAD

**Issue by mbtools** [abaplint/abaplint#1400]
I suggest that the check should not raise errors, if 
- there are multiple parameters of the same kind - for example only 'importing' parameters
- the method call is used as an expression - for example in an if statement

Example:
```
* Bad example
method( IMPORTING parameter1 = value EXPORTING parameter2 = value ).

* Good example
method(
  IMPORTING
    parameter1 = value 
  EXPORTING 
    parameter2 = value ).
   
* Good example
method( parameter1 = value parameter2 = value ).

* Good example
IF method( IMPORTING parameter1 = value EXPORTING parameter2 = value ).
...
ENDIF.
```

**Comment by sbcgua** [comment]
To add 5 cents ...
One of my favorite videos. https://www.youtube.com/watch?v=ZsHMHukIlJY
The subject is clarified staring at 10:30 ~ 23:00 with some very bright examples at ~21:00+
In a nut shell: https://github.com/SAP/styleguides/issues/21 and comments
So imho `method( parameter1 = value parameter2 = value ).` is not good as the second param is not visually distinguished and is at random position.

P.S. though maybe I'd add an exception for methods with very short param names like built-in string functions ... :)

**Comment by larshp** [comment]
In the default setup it should be as strict as possible, then we can add additional configuration to allow more stuff

**Comment by mbtools** [comment]
excellent video indeed. one of the key points is about line length and length of identifiers. the shorter and the more meaningful, the better. And when you do that imho you don't need to break things into multiple lines. 

`my_biggest_asset = max( val1 = my_car val2 = my_boat val3 = my_house ).`

`if max( val1 = my_car val2 = my_boat ) > min( val1 = my_house val2 = my_chateau ).`

The rule makes sense but it should provide some flexibility like
- Built-ins vs class method
- direct method call/returning assignment vs expression that's part of another statement
- Single type of parameter (just imp/exp/etc) vs mixed

**Comment by sbcgua** [comment]
Maybe the criteria is short key/value pairs ? E.g. `a = 1 b = 'abc'` is still readable, while `iv_a_param = 1 iv_b_argument = 'abc'` is already not. (just thoughts)

**Comment by mbtools** [comment]
Number of params is configurable with #2076
